index.html

```diff
@@ -253,9 +253,9 @@ <h2>History of <strong class="theme_blue">handheld</strong></h2>
                         <div class="col-lg-12 col-md-12 col-sm-12">
                             <h3>Team member</h3>
                             <p>
-                                Daniel Gebreselassie &nbsp; &nbsp; &nbsp; &nbsp; Donnovin Rogers &nbsp; &nbsp; &nbsp; &nbsp; e Brink &nbsp; &nbsp; &nbsp; &nbsp; Jeremy Robinson
+                                Daniel Gebreselassie &nbsp; &nbsp; &nbsp; &nbsp; Donnovin Rogers &nbsp; &nbsp; &nbsp; &nbsp; Emily Brink &nbsp; &nbsp; &nbsp; &nbsp; Jeremy Robinson

```

Comment:
@brinkbrink how would you like your name here?

Reply:
Never mind on this.  Probably not a big deal.

Reply:
E. Brink, I'll change it